example/ck_tile/38_block_scale_gemm/gemm_utils.hpp

```diff
@@ -11,12 +11,16 @@
 #include "ck_tile/ops/gemm.hpp"
 #include "ck_tile/ops/gemm_quant.hpp"
 
-#define CK_TILE_SUPPORTED_QUANT_GROUPS(X) \
-    X(1, 1, 64)   /* 1D */                \
-    X(1, 1, 128)  /* 1D */                \
-    X(1, 8, 128)  /* 2D N=8  */           \
-    X(1, 32, 128) /* 2D N=32 */           \
-    X(1, 64, 128) /* 2D N=64 */
+inline size_t hash_multiple_strings(const std::vector<std::string>& inputs)
+{
+    std::hash<std::string> hasher;
+    size_t combined_hash = 0;
+    for(const auto& str : inputs)
+    {
+        combined_hash ^= hasher(str) + 0x9e3779b9 + (combined_hash << 6) + (combined_hash >> 2);
```

Comment:
I had to google this (nice trick!), let us add a comment to make it more obvious.

```suggestion
        // Hash combine using golden ratio constant and bit shifts for good distribution and order-dependent mixing
        combined_hash ^= hasher(str) + 0x9e3779b9 + (combined_hash << 6) + (combined_hash >> 2);
```

Reply:
done